build.gradle

```diff
@@ -140,7 +140,7 @@ repositories {
     mavenCentral()
     maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
     maven { url "https://plugins.gradle.org/m2/" }
-    maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
+    maven { url "https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/" }
```

Comment:
Are we publishing `SNAPSHOTS` artifacts to this one?

Reply:
Nope. This is for fetching Lucene related artifacts.